test.js

```diff
@@ -126,6 +126,30 @@ test('`configName` option', t => {
 	t.is(path.basename(conf.path, '.json'), configName);
 });
 
+test('no `suffix` option', t => {
+	const conf = new Conf();
+	// Maintain behaviour of 2.0.0
+	t.true(conf.path.includes('-node'));
+});
+
+test('with `suffix` option set to empty string', t => {
+	const projectSuffix = '';
+	const conf = new Conf({projectSuffix});
+	// Config file's directory root
+	const configRoot = path.resolve(conf.path, '../..');
+	// Extract the config file's parent folder
+	const rootName = path.basename(configRoot);
```

Comment:
You should also assert that the `rootName` is correct, as that will only pass on Windows currently.

Reply:
Shouldn't test executed on Travis be meant to reflect actual behavior on Linux and Mac as well? I will add it nonetheless if it makes tests more future-proof.

Reply:
I think you're misunderstanding what I'm saying. The test is currently faulthy as `t.true(conf.path.includes(`-${projectSuffix}`));` will always pass, since it tests the a negative case but never asserts that what it tests is correct.

You will also have to adjust `const configRoot = path.resolve(conf.path, '../..');` for macOS and Linux.